### PR TITLE
init models and relationships

### PR DESCRIPTION
**Pull request by nguyenquanglinh2898**
## Related Tickets:
- [#30159](https://edu-redmine.sun-asterisk.vn/issues/30159)

## WHAT this PR does?
- Tạo models và relationships

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by nguyenquanglinh2898** [comment]
ready

File: app/Models/Comment.php [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    {
+        return $this->belongsTo(User::class);
+    }
+
+    public function product()
+    {
+        return $this->belongsTo(Product::class);
+    }
+
+    public function images( )
```
Comment (huyenttm-0901) [inline app/Models/Comment.php]:
```suggestion
    public function images()
```

File: app/Models/User.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+use Illuminate\Contracts\Auth\MustVerifyEmail;
+use Illuminate\Foundation\Auth\User as Authenticatable;
+use Illuminate\Notifications\Notifiable;
+
+class User extends Authenticatable
+{
+    use Notifiable;
+
+    protected $fillable = [
+        'name', 'email', 'password',
```
Comment (huyenttm-0901) [inline app/Models/User.php]: uống dòng đi em

**Comment by nguyenquanglinh2898** [comment]
ready

**Review by huyenttm-0901 — COMMENTED** [review]
conflicted

**Comment by nguyenquanglinh2898** [comment]
ready

**Pull request merged**